google/cloud/bigtable/tests/instance_admin_emulator.cc

```diff
@@ -38,9 +38,10 @@ class InstanceAdminEmulator final
       grpc::ServerContext* context,
       btadmin::CreateInstanceRequest const* request,
       google::longrunning::Operation* response) override {
+    std::unique_lock<std::mutex> lk(mu_);
```

Comment:
What is the lock actually protecting?  The suggestion is that its scope be minimized (and particularly to not hold it over the cout streaming).

Reply:
I am just trying to make the output readable. If we do not protect the `std::cout` we get messages interleaved from different calls.

Beyond that, the data structures in this class need protection, of course.

Reply:
OK.